src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -1574,7 +1581,7 @@ def __init__(
                     num_attention_heads=temporal_num_attention_heads,
                     in_channels=out_channels,
                     num_layers=temporal_transformer_layers_per_block[i],
-                    norm_num_groups=temporal_norm_num_groups,
```

Comment:
Nice catch here. Can we just remove the `temporal_norm_num_groups` argument from the `UpBlockMotion` class

Reply:
Let's do this in #8846?